### PR TITLE
Add april 2022 meeting details

### PR DESCRIPTION
**Pull request by jabawack81**

### CODE REVIEW

File: source/meetings/2022/april/index.html.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+### Using React in a Ruby Project to Dynamically Generate PDF Documents
+
+[Panos Matsinopoulos](https://getlavanda.com) says:
+
+
+> Fjust needs to be a couple of sentences really, not a
+> full abstract. We use markdown so feel free to format it or embed links for
+> the writeup.
+>
```
Comment (proserpioerena) [inline source/meetings/2022/april/index.html.md:32]: I believe you may have forgotten to delete these lines. May you confirm?

File: source/meetings/2022/april/index.html.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+> or libraries like [whtmltopdf](https://wkhtmltopdf.org/).
+> 
+> Recently, in one of our Ruby on Rails projects in which we 
+> wanted to generate PDF documents for invoices, we decided to use
+> another programming language and technology: React and AWS Lambda.
+> 
+> In this talk, we will be covering how we did it, what were the challenges
+> and what pros and cons over the incumbent tools for Ruby.
+
+{::coverage year="2022" month="april" talk="using-react-in-a-ruby-project-to-dynamically-generate-p-d-f-documents" /}
```
Comment (proserpioerena) [inline source/meetings/2022/april/index.html.md:44]: A bit of a nitpick, but is it desirable to spell `pdf` as `p-d-f`?

File: source/meetings/2022/april/index.html.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### A parse parse pitch: using JSON and custom parsers to create efficient flexible data structures.
+
+[Eleanor McHugh](http://slideshare.net/feyeleanor) says:
+
+> JSON and custom parsers let you do neat things like: 
+> complex pricing, map data to 3D models, auto generate images… 
+
+{::coverage year="2022" month="march" talk="a-parse-parse-pitch-using-j-s-o-n-and-custom-parsers-to-create-efficient-flexible-data-structures" /}
```
Comment (proserpioerena) [inline source/meetings/2022/april/index.html.md:53]: Another nitpick here: are you comfortable with having `json` as `j-s-o-n`? As for the above, it is more about usability on our end when we add the coverage with the videos etc.

File: source/meetings/2022/april/index.html.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Even in a virtual world there are limited places for attending the
+meeting so **[you need to register via eventbrite][apr2022-eventbrite]**.
+The link to the zoom meeting will only be sent to registered attendees on
+the day of the meeting, around about 6pm. Keep an eye out for the email
+and check your spam just in case.
+
+Do prepare yourself in advance by [downloading the zoom
+client](https://zoom.us/support/download).
+
+[apr2022-eventbrite]: https://www.eventbrite.com/e/lrug-march-2022-tickets-292085996077
```
Comment (proserpioerena) [inline source/meetings/2022/april/index.html.md:82]: Link points to the March event.

File: source/meetings/2022/april/index.html.md (hunk 1 above)
Comment (jabawack81) [reply source/meetings/2022/april/index.html.md:32]: yeah overly eager copypasta

File: source/meetings/2022/april/index.html.md (hunk 2 above)
Comment (jabawack81) [reply source/meetings/2022/april/index.html.md:44]: I've converted the case of the sentence with sublime text and it does that

File: source/meetings/2022/april/index.html.md (hunk 3 above)
Comment (jabawack81) [reply source/meetings/2022/april/index.html.md:53]: I've converted the case of the sentence with sublime text and it does that

File: source/meetings/2022/april/index.html.md (hunk 4 above)
Comment (jabawack81) [reply source/meetings/2022/april/index.html.md:82]: I still haven't published the event on Eventbrite I'll update once that is done

**Pull request merged**